src/DevBetterWeb.Core/Interfaces/IWebhookHandlerService.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Threading.Tasks;
+
+namespace DevBetterWeb.Core.Interfaces
+{
+  public interface IWebhookHandlerService
```

Comment:
Another service with 4 methods that should just be 1 method and a billing action type (subscribe/renew/cancel/end)